roles/setup_etcd/tasks/setup_etcd.yml

```diff
@@ -64,7 +68,7 @@
 - name: Prepare etcd_inital_cluster based on private_ip
   ansible.builtin.set_fact:
     etcd_initial_cluster: >-
-      {{ etcd_initial_cluster | default([]) + [node.inventory_hostname + '=http://' + node.private_ip + ':' + etcd_peer_port | string] }}
+      {{ etcd_initial_cluster | default([]) + [node.inventory_hostname + http_type + node.private_ip + ':' + etcd_peer_port | string] }}
```

Comment:
Should this also be `_http_type`?

Reply:
Yes. I cannot see this in PR. Maybe you are looking at the old commit.

Reply:
Nevermind. Found it. Fixed.